### ISSUE TITLE
Key bindings does not work in yakindu sct text editors

### ISSUE THREAD

**Issue by wohaug** [Yakindu/statecharts#2081]
Create a new state chart from the examples (e.g. the traffic light example that comes with yakindu sct).
Try to use common key accelerators within any embedded text editor in canvas or in the text editor of properties view. Keys accelerators do not work.
I tried for example: "Ctrl+D" to delete a line or "Alt+up" / "Alt+down" to move a line.

In preference dialog General->Keys those key bindings appear, so I expect that they should work.

Note: I wonder that I cannot find any caller for IContextService.activateContext in yakindu sct code.

**Comment by tkutz** [comment]
Yes, you are right, "Ctrl-D" and "Alt-up/down" do not work in most cases right now.

It works for the statechart definition section in pinned mode, though.
